utils/test_harness/tools/src/test_harness_metric.cpp

```diff
@@ -252,6 +252,8 @@ std::vector<metricGroupInfo_t> optimize_metric_group_info_list(
 
   LOG_INFO << "size of optimizedList based on percentage "
            << optimizedList.size();
+  EXPECT_GT(optimizedList.size(), 0u)
```

Comment:
I suspect the test will not fail if the EXPECT() calls are in the harness, please confirm. If so, makes no sense to do it here.

Reply:
Non-fatal failures. Removed, Thank you.